dwave/preprocessing/include/dwave/presolve.h

```diff
@@ -344,12 +344,22 @@ class Presolver {
         bool ret = false;
 
         ret |= remove_zero_biases(model_.objective);
-        for (size_t c = 0; c < model_.num_constraints(); ++c) {
+        for (size_type c = 0; c < model_.num_constraints(); ++c) {
             ret |= remove_zero_biases(model_.constraint_ref(c));
         }
 
         return ret;
     }
+    bool technique_remove_small_biases() {
+        bool ret = false;
+
+        for (size_type c = 0; c < model_.num_constraints(); ++c) {
+            if (model_.constraint_ref(c).is_linear())
```

Comment:
I think this check should actually happen in `remove_small_biases`. Otherwise a user using that function directly might get surprising behavior.

Reply:
Makes sense. Done.